web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/beregningsgrunnlag/BeregningsgrunnlagOverstyringshåndterer.java

```diff
@@ -54,24 +54,32 @@ public class BeregningsgrunnlagOverstyringshåndterer extends AbstractOverstyrin
     public OppdateringResultat håndterOverstyring(OverstyrBeregningsgrunnlagDto dto,
                                                   Behandling behandling, BehandlingskontrollKontekst kontekst) {
         var tjeneste = beregningsgrunnlagInputTjeneste.getTjeneste(behandling.getFagsakYtelseType());
-        var input = tjeneste.lagInput(BehandlingReferanse.fra(behandling));
+        var ref = BehandlingReferanse.fra(behandling);
+        var input = tjeneste.lagInput(ref);
         beregningHåndterer.håndterBeregningsgrunnlagOverstyring(input, OppdatererDtoMapper.mapOverstyrBeregningsgrunnlagDto(dto));
-        // Lag historikk
+
+        lagHistorikk(ref, dto);
+
         var builder = OppdateringResultat.utenTransisjon();
         fjernOverstyrtAksjonspunkt(behandling)
             .ifPresent(ap -> builder.medEkstraAksjonspunktResultat(ap.getAksjonspunktDefinisjon(), AksjonspunktStatus.AVBRUTT));
         return builder.build();
     }
 
+    @Override
+    protected void lagHistorikkInnslag(Behandling behandling, OverstyrBeregningsgrunnlagDto dto) {
+        // Håndteres sammen med overstyringen
```

Comment:
Alle overstyringer gjør dette sentralt - se aksjonspunkttjeneste + AbstractOverstyringshåndterer
Du bør teste ut dette lokalt og se oppførselen.
Skal vi endre pattern for overstyring bør vi gjøre det overalt og etter litt tenketid - framfor lokale varianter.